frame/pablo/src/lib.rs

```diff
@@ -1423,6 +1519,10 @@ pub mod pallet {
 			lp_amount: Self::Balance,
 			min_amount: Self::Balance,
 		) -> Result<(), DispatchError> {
+			ensure!(
+				AccountsDepositedOneAsset::<T>::get(who, &pool_id) >= lp_amount,
```

Comment:
And what if, for example, a user has deposited liquidity in two assets, and wants to withdraw it in one asset?

Reply:
He can't do like this. He can withdraw with one asset only if he deposited with one asset.

Reply:
Sure. Resolved!